syntax = "proto3";

option java_package = "io.grpc.examples";

package helloworld;

//问候服务定义
service Greeter {
        //发送消息
        rpc SayHello(HelloRequest) returns (HelloReply){}
}

//请求消息包含name
message HelloRequest {
        string name = 1;
}

//响应消息包含message
message HelloReply {
        string message = 1;
}
